solutions/cses-1081.mdx

```diff
@@ -44,6 +46,55 @@ int main() {
 	cout << ans << endl;
 }
 ```
+</CPPSection>
+
+<JavaSection>
+```java
+import java.util.Scanner;
+
+public class Main {
```

Comment:
could you rename this to some form of the problem name

Reply:
fixed